packages/headless/src/api/search/facet-search/base/base-facet-search-request.ts

```diff
@@ -5,7 +5,7 @@ export interface FacetSearchRequestOptions {
   /** A dictionary that maps index field values to facet value display names. */
   captions: Record<string, string>;
   /** The maximum number of values to fetch.
-   * @default 10
+   * @defaultValue `10`
```

Comment:
One thought about this change. Even though `defaultValue` is the tag supported by api-extractor, I noticed that vscode applies special highlighting only to the `default` tag, for numbers, strings, booleans.

I thought the highlighting was a nice touch when hovering over. Would it be possible to adjust the parser somehow to support `default` instead?

Reply:
I tried to add a custom tag to the api extractor the other day and it's a dirty workaround, since custom tags essentially just get added as text to whichever tag is above them.

Reply:
Exactly as @btaillon said, I also tried adding a custom tag, and the only way to do it where it becomes its own tag instead of getting added as text to the tag above it, is an unsupported workaround that I couldn't even get fully working.  If custom tags ever work later, we can very easily switch back to `default` tags at the same time as switching back to `minimum` tags.